### PR TITLE
Rid of description fields in boolean and object to get rid of repeate…

### LINKED ISSUE #84: Booleans repeat the description
image

We need to override booleans so we aren't repeating data

### PR DESCRIPTION
**Pull request by uppal101**
…d description. Fix #84

### CODE REVIEW

**Comment by uppal101** [comment]
I left the extra schemas in there for later since we addressed at some point array and object description repetition will need to be fixed

File: packages/api-explorer/__tests__/form-components/CheckboxWidget.test.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const checkbox = shallow(
+    <Checkbox
+      id="id"
+      label="label"
+      required
+      onChange={() => {}}
+      schema={{ description: 'description' }}
+    />,
+  );
+  expect(checkbox.html()).not.toMatch(/description/);
```
Comment (uppal101) [inline packages/api-explorer/__tests__/form-components/CheckboxWidget.test.jsx]: Question the .not.toMatch(/description/) means the string description shouldn't show up in the snapshot correct?

File: packages/api-explorer/src/form-components/ObjectField.jsx [hunk 2]
```diff
@@ -55,13 +55,6 @@ class CustomObjectField extends ObjectField {
             formContext={formContext}
           />
         )}
-        {(uiSchema['ui:description'] || schema.description) && (
```
Comment (uppal101) [inline packages/api-explorer/src/form-components/ObjectField.jsx]: This change came from the original directory I had locally I think we can keep it as we discussed until we figure out how to design readme to be recursive?

File: example/swagger-files/oas.json [hunk 3]
```diff
@@ -0,0 +1,274 @@
+{
```
Comment (uppal101) [inline example/swagger-files/oas.json]: should this file be deleted?

File: packages/api-explorer/__tests__/form-components/CheckboxWidget.test.jsx (hunk 1 above)
Comment (domharrington) [reply packages/api-explorer/__tests__/form-components/CheckboxWidget.test.jsx]: I removed the snapshot test. It was simple enough to do it without snapshots

File: packages/api-explorer/src/form-components/ObjectField.jsx (hunk 2 above)
Comment (domharrington) [reply packages/api-explorer/src/form-components/ObjectField.jsx]: Ahhh yeah i think we should revert this change until we've fully figured out what the repercussions are and fixed that issue separarely?

**Pull request merged**